### ISSUE TITLE
`docker run -v` doesn't like a relative path for `host-src`

### ISSUE THREAD

**Issue by heskew** [apparatus/xeno-compose#5]
I have a docker container that's configured to mount a couple of volumes pointing to child dirs. Those paths need to remain absolute for use with `docker run` otherwise the host paths will be seen as invalid names. 

Here's the relevant bit in question:
https://github.com/apparatus/xeno-compose/blob/master/compose.js#L119

Is there a reason to keep the host path relative? If I update that bit to set `v[0]` to `resolved` then the `docker run` command works fine.

**Comment by AdrianRossouw** [comment]
it's completely possible that this is just a mistake on my part.

We've been keeping all our paths relative since fuge is a development tool, hence we are avoiding hardcoding paths on developers machines.

thus, the pathways that support absolute parts aren't as well tested as relative paths or data containers.

**Comment by heskew** [comment]
Yeah, relative paths totally make sense for dev. Just that docker run cli volume setup is making that difficult. I'll try a workaround or two over the weekend before thinking about a pr.

**Comment by AdrianRossouw** [comment]
it should be really simple, just in those few lines. i just don't have the time to check myself

**Comment by heskew** [comment]
A fix is simple, haven't tried adding tests for this specific change though.

https://github.com/apparatus/xeno-compose/pull/6
